dist/pom.xml

```
@@ -124,6 +124,17 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>tasklist-webapp</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>commons-logging</groupId>
+          <artifactId>commons-logging</artifactId>
+        </exclusion>
```

```suggestion
        </exclusion>
        <exclusion>
          <groupId>org.springframework</groupId>
          <artifactId>spring-webflux</artifactId>
        </exclusion>
```

With graphql, Tasklist includes spring-webflux transitively. Would be great to not include this one in the distribution.

---

Just did a small test locally, and it didn't cause any issues with excluding

---

I would better exclude it form `com.graphql-java-kickstart:graphql-spring-boot-autoconfigure` in dependency management

---

Fixed here https://github.com/camunda/zeebe/pull/17649/commits/69aff526c8125352b9617d9b2b34930466ae1311